public/schema.json

```diff
@@ -2369,6 +2369,20 @@
               "isDeprecated": false,
               "deprecationReason": null
             },
+            {
+              "name": "genericItemsCount",
```

Comment:
das hier nicht zu surveys dazugehörige im schema dump hab ich immer weggelassen...aaaber ok, dann isses jetz ab sofort mit drin

Reply:
aber generis_item ist doch schon lange drin in master, wieso fehlt denn dann das Schema dazu?

Reply:
wurde damals vergessen, denk ich

Reply:
sauber wäre es im master nachzuziehen und dann hier nicht...aber komme mit klar, wenn es dann jetzt so ist und bleibt